src/OrbitGl/Viewport.cpp

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "Viewport.h"
+
+#include <algorithm>
+
+#include "OrbitBase/Logging.h"
+
+namespace orbit_gl {
+
+Viewport::Viewport(int width, int height) : width_(width), height_(height) {
```

Comment:
Nit: also `world_width_(width), world_height_(height)`?

Reply:
And extents_, actually :-) Changed.